src/main/java/uk/gov/hmcts/cmc/claimstore/models/Claim.java

```diff
@@ -125,36 +141,38 @@ public LocalDateTime getCountyCourtJudgmentRequestedAt() {
     }
 
     @Override
-    @SuppressWarnings("squid:S1067") // Its generated code for equals sonar
-    public boolean equals(Object other) {
-        if (this == other) {
+    public boolean equals(Object obj) {
+        if (this == obj) {
             return true;
         }
-        if (other == null || getClass() != other.getClass()) {
+        if (obj == null || getClass() != obj.getClass()) {
             return false;
         }
-        Claim otherClaim = (Claim) other;
-        return Objects.equals(id, otherClaim.id)
-            && Objects.equals(submitterId, otherClaim.submitterId)
-            && Objects.equals(submitterEmail, otherClaim.submitterEmail)
-            && Objects.equals(letterHolderId, otherClaim.letterHolderId)
-            && Objects.equals(defendantId, otherClaim.defendantId)
-            && Objects.equals(externalId, otherClaim.externalId)
-            && Objects.equals(referenceNumber, otherClaim.referenceNumber)
-            && Objects.equals(claimData, otherClaim.claimData)
-            && Objects.equals(createdAt, otherClaim.createdAt)
-            && Objects.equals(issuedOn, otherClaim.issuedOn)
-            && Objects.equals(responseDeadline, otherClaim.responseDeadline)
-            && Objects.equals(respondedAt, otherClaim.respondedAt)
-            && Objects.equals(countyCourtJudgment, otherClaim.countyCourtJudgment)
-            && Objects.equals(countyCourtJudgmentRequestedAt, otherClaim.countyCourtJudgmentRequestedAt);
+        Claim claim = (Claim) obj;
+        return moreTimeRequested == claim.moreTimeRequested
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (16) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1067)